pyEpiabm/pyEpiabm/sweep/travel_sweep.py

```diff
@@ -0,0 +1,239 @@
+            "prob_exposed_to_asympt"]
+        if Parameters.instance().use_ages:
+            # Age used in model
+            age_prop = Parameters.instance().age_proportions
+            # travellers are between 15-80 years
+            age_prop_adjusted = [0.0 if i in [0, 1, 2, 16] else prop for
+                                 i, prop in enumerate(age_prop)]
+            w = age_prop_adjusted / sum(age_prop_adjusted)
+            microcell_split = np.random.multinomial(
+                number_individuals_introduced, w, size=1)[0]
+            for age in range(len(age_prop_adjusted)):
+                number_indiv_agegroup = microcell_split[age]
+                number_indiv_agegroup_InfectedAsympt = \
+                    math.floor(asymp_prop[age] * number_indiv_agegroup)
+                number_indiv_agegroup_InfectedMild = \
+                    number_indiv_agegroup - \
+                    number_indiv_agegroup_InfectedAsympt
+                self.initial_microcell.add_people(
+                    number_indiv_agegroup_InfectedAsympt,
+                    status=InfectionStatus.InfectASympt,
+                    age_group=age)
+                self.initial_microcell.add_people(
+                    number_indiv_agegroup_InfectedMild,
+                    status=InfectionStatus.InfectMild,
+                    age_group=age)
+        else:
+            # Age not used in model
+            if isinstance(asymp_prop, list):
+                asymp_prop = asymp_prop[0]
+            number_indiv_InfectedAsympt = \
+                math.floor(asymp_prop * number_individuals_introduced)
+            number_indiv_InfectedMild = number_individuals_introduced - \
+                number_indiv_InfectedAsympt
+            self.initial_microcell.add_people(
+                    number_indiv_InfectedAsympt,
+                    status=InfectionStatus.InfectASympt)
+            self.initial_microcell.add_people(
+                    number_indiv_InfectedMild,
+                    status=InfectionStatus.InfectMild)
+
+        for person in self.initial_cell.persons:
+            # Assign introduced individuals a time to stay
+            person.travel_end_time = time + random.randint(2, 14)
+            # Assigns them their next infection status and the time of
+            # their next status change. Also updates their infectiousness.
+            person.next_infection_status = InfectionStatus.Recovered
+            HostProgressionSweep.set_infectiousness(person, time)
+            HostProgressionSweep().update_time_status_change(person, time)
+
+    def assign_microcell_household(self, number_individuals_introduced):
+        """
+        Assign introduced individuals to microcells based on population
+        density of micorcells. Take a number of microcells equal to the
+        the number_individuals_introduced (or max number of microcells)
+        and assign individuals randomly to one of these microcells,
```

Comment:
```suggestion
        density of micorcells. Takes a number of microcells equal to the
        the number_individuals_introduced (or max number of microcells)
        and assigns individuals randomly to one of these microcells,
```

This is a description of functionality, not an instruction

Reply:
Good spot, done